tests/test_datasets/test_internal_datasets.py

```diff
@@ -79,7 +79,7 @@ def test_not_present_part():
 
 
 @pytest.mark.parametrize(
-    "dataset_name, expected_shape, expected_min_date, expected_max_date, dataset_parts",
+    "dataset_name, expected_shape, expected_min_idx, expected_max_idx, dataset_parts",
```

Comment:
Why `expected_min_idx`, not `expected_min_timestamp`?

Reply:
Okey, I will replace it

Reply:
I fixed it.